### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by philippemiron** [OceanParcels/plasticparcels#51]
Hi all,

Would it be possible to include at least ~1-2 examples where all the input datasets are publicly available? 

More precisely, after installing `plasticparcels`, I can't reproduce any of the examples, as e.g. [example_Italy_coast](https://plastic.oceanparcels.org/en/latest/examples/example_Italy_coast.html)  requires the model settings:

```
# Load the model settings
settings_file = 'docs/examples/example_Italy_coast_settings.json'
settings = pp.utils.load_settings(settings_file)
```
which are not part of the path. 

Also, looking at those example configuration files [in the repo](https://github.com/OceanParcels/plasticparcels/tree/main/docs/examples), they all rely on private storage `/storage/shared/oceanparcels/input_data/`.

openjournals/joss-reviews#7094

**Comment by michaeldenes** [comment]
Hi @philippemiron,

You've raised a good point, to run `plasticparcels` requires some hydrodynamic model data, biogeochemical model data, as well as wind and wave data. The example notebooks all use this model data, hence the storage filepath `/storage/shared/oceanparcels/input_data/`. We'll make this more clear in the `plasticparcels` documentation. We will, however, provide an example using an idealised fields that can be run independent of the required datasets.

In essence, `plasticparcels` can be run on any dataset that `parcels` can be run on, however we have configured it for Copernicus Marine Service data for now. Our near-future goal is to release `plasticparcels` on a cloud-platform, ideally in an environment with a variety of available hydrodynamic and biogeochemical datasets.

Expect to see an idealised example to try out soon!

**Comment by philippemiron** [comment]
I think that will be very helpful to test out the kernels, and will also aligned a bit more with the rest of the parcel documentation.

**Comment by michaeldenes** [comment]
Hi @philippemiron, we've now created an idealised example that requires no additional data (see #55), and have updated the documentation for all the other examples. We'll roll these out in a new release soon, but they are all in the main branch now. Let me know if you have any issues, feel free to close this issue if all works! Cheers!

**Comment by philippemiron** [comment]
I just reviewed the example and it is great, but I had a problem with the `conda-forge` package on two machines (intel osx and arm64 osx) running the first cell:

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 7
      4 import numpy as np
      6 # parcels and plasticparcels imports
----> 7 import plasticparcels as pp
      8 import parcels
     10 # Plotting imports

File [~/micromamba/envs/test/lib/python3.12/site-packages/plasticparcels/__init__.py:5](http://localhost:8888/lab/tree/Downloads/plasticparcels/docs/examples/~/micromamba/envs/test/lib/python3.12/site-packages/plasticparcels/__init__.py#line=4)
      2 from plasticparcels.particlefile import *  # noqa
      3 from plasticparcels.utils import *  # noqa
----> 5 from plasticparcels._version_setup import __version__, __version_tuple__  # noqa

ModuleNotFoundError: No module named 'plasticparcels._version_setup'
```
It worked once I installed the current main branch of `plasticparcels` using `pip`... I will only recommend we use the next version of `plasticparcels` alongside the JOSS paper so the example is included.

**Comment by philippemiron** [comment]
Small comment, in the last cell where you are comparing the 5 experiments, I think `ax.set_ylabel('Depth [m]')` should be `ax.set_ylabel('Meridional distance [m]')`.

**Comment by michaeldenes** [comment]
Thanks for raising the versioning issue @philippemiron, this should be fixed by #59, and I've updated the y-axis labelling in #58. Cheers!
